tests/filecheck/dialects/ltl/ltl_op.mlir

```diff
@@ -0,0 +1,19 @@
+// RUN XDSL_ROUNDTRIP
+
+"builtin.module"() ({
+    %b = "arith.constant"() {value = 0} : () -> i1
+    %seq = "arith.constant"() {value = 0} : () -> !ltl.sequence
+    %p = "arith.constant"() {value = 0} : () -> !ltl.property
+    "ltl.and"(%seq, %seq) : (!ltl.sequence,!ltl.sequence)->(!ltl.sequence)
+    "ltl.and"(%p, %p) : (!ltl.property,!ltl.property)->(!ltl.property)
+    "ltl.and"(%b, %b) : (i1,i1)->(i1)
+}) : () -> ()
```

Comment:
```suggestion
builtin.module {
    %b, %seq, %p = "test.op"() : () -> (i1, !ltl.sequence, !ltl.property)
    "ltl.and"(%seq, %seq) : (!ltl.sequence,!ltl.sequence)->(!ltl.sequence)
    "ltl.and"(%p, %p) : (!ltl.property,!ltl.property)->(!ltl.property)
    "ltl.and"(%b, %b) : (i1,i1)->(i1)
}) : () -> ()
```
We try to use test ops for operations where the source of the SSA value is not directly relevant to the test. In this case, arith is not being tested, so we wouldn't want this test to fail if we break arith.

Reply:
Similarly in other test

Reply:
oh right! makes absolutely sense, correcting it rn